pkg/reporting/app.go

```diff
@@ -325,3 +361,42 @@ func getGitOpsReport(clientset kubernetes.Interface, appID string, clusterID str
 	}
 	return false, ""
 }
+
+func getSnapshotReport(kotsStore store.Store, clientset kubernetes.Interface, veleroClient veleroclientv1.VeleroV1Interface, appID string, clusterID string) (*SnapshotReport, error) {
+	report := &SnapshotReport{}
+
+	bsl, err := snapshot.FindBackupStoreLocation(context.TODO(), clientset, veleroClient, util.PodNamespace)
```

Comment:
How about calling this outside of the `getSnapshotReport` function and pass the BSL as a parameter to avoid refactoring lots of the code?

Reply:
bsl as a parameter to `getSnapshotReport` makes sense. it still seems like a worthwhile refactor to open up unit testing of snapshot code paths, but i can revert that if needed